packages/store/src/configs/messages.config.ts

```diff
@@ -21,8 +21,8 @@ export const CONFIG_MESSAGES: ObjectKeyMap<Function> = {
     `State name '${current}' from ${newName} already exists in ${oldName}`,
   [VALIDATION_CODE.STATE_DECORATOR]: () => 'States must be decorated with @State() decorator',
   [VALIDATION_CODE.INCORRECT_PRODUCTION]: () =>
-    'Angular is running in production mode but NGXS is still running in the development mode!\n' +
-    'Please set developmentMode to false on the NgxsModule options when in production mode.\n' +
+    'Angular is running in the production mode but NGXS is still running in the development mode!\n' +
+    'Please set developmentMode to false on the NgxsModule options in the production mode.\n' +
```

Comment:
Revert the changes to both lines, it was correct before.

Reply:
Oh, and `in the development mode` should rather be `in development mode`

Reply:
Not a native speaker and cannot argue, just saw such statement in Angular's code:
![image](https://user-images.githubusercontent.com/7337691/63888771-4b093100-c9e8-11e9-90e8-dbd5c1805a7a.png)
But no problem for sure!

Reply:
Done!